web/modules/custom/usagov_ssg_postprocessing/src/Controller/ssgStatController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\usagov_ssg_postprocessing\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * An usagov_ssg_postprocessing controller for stating the current static-site-gen status.
+ */
+class ssgStatController extends ControllerBase {
+
+    public function content() {
+
+        $date = \Drupal::state()->get('ssg_stat_date');
+        $msg = \Drupal::state()->get('ssg_stat_msg');
+
+        if ( empty($msg) ) {
+            $markup = "Static Site Generator has not been run on this environment yet.";
+        } else {
+            date_default_timezone_set('America/New_York');
+            $formatDate = date('D F jS Y, h:ia', $date);
```

Comment:
Please add "T" to the end of the format (e.g., 'D F jS Y, h:ia T' and remove "EST" from the markup on the next line. (This is the benefit I was looking for in moving from shell to something else -- you can get the correct timezone automatically.)

Reply:
ok, this has been fixed